python/common/thief_constants.py

```diff
@@ -93,3 +93,52 @@ class Events(object):
     SENDING_PAIRING_REQUEST = "SendingPairingRequest"
     RECEIVED_PAIRING_RESPONSE = "ReceivedPairingResponse"
     PAIRING_COMPLETE = "PairingComplete"
+
+
+class MetricNames(object):
+    # ---------------------
+    # System Health metrics
+    # ---------------------
+    PROCESS_CPU_PERCENT = "processCpuPercent"
```

Comment:
These should be PascalCased per the spec, right?

Reply:
Yes.  Metrics are PascalCased when pushed to Azure Monitor and camelCased when included in JSON.